### DISCUSSION TITLE
Remix preset: accessing files in the public dir

### DISCUSSION THREAD

**Discussion by trurl-master** [vercel/community#328] [Help]
Hello!

I'm having trouble accessing files in the public directory, I tried using path resolve, like so: `path.resolve("public/locales")`. It works perfectly fine with `vercel dev` but not at all when deployed. What is the best way to access them?

Thank you!

**Comment by leerob** [comment]
Related discussion: https://github.com/vercel/community/discussions/229

**Reply by trurl-master** [reply]
> Related discussion: #229

I tried using `includeFiles`, but with no success, the files actually don't get copied, I reference them like this (the `locales` folder is put next to the `api/index`):

```json
{
  "build": {
    "env": {
      "ENABLE_FILE_SYSTEM_API": "1"
    }
  },
  "functions": {
    "api/index.js": {
      "includeFiles": "locales/**"
    }
  }
}
```

When putting files in `public`, they do get copied:

<img width="166" alt="Screenshot 2022-02-13 at 18 50 29" src="https://user-images.githubusercontent.com/640657/153767799-94e54781-efe4-4744-88da-81649ac99a84.png">

But I have no idea how to reference them. Relative path just give me "no such file" '/var/task/output/server/pages/locales/en/common.json'. Where inside this tree is `locales`?
